### PR TITLE
Fixes #29003 - Fix controller path for task search bookmarks

### PR DESCRIPTION
**Pull request by adamruzicka**

### CODE REVIEW

File: webpack/ForemanTasks/Components/TasksTable/TasksTableConstants.js [hunk 1]
```diff
@@ -1,7 +1,7 @@
 import { getControllerSearchProps } from 'foremanReact/constants';
 
 export const TASKS_TABLE_ID = 'TASKS_TABLE';
-export const TASKS_CONTROLLER = 'tasks';
+export const TASKS_CONTROLLER = 'foreman_tasks/tasks';
```
Comment (adamruzicka) [inline webpack/ForemanTasks/Components/TasksTable/TasksTableConstants.js]: Quick grep says this isn't being used anywhere. Should we keep it around?

File: webpack/ForemanTasks/Components/TasksTable/TasksTableConstants.js [hunk 2]
```diff
@@ -1,7 +1,7 @@
...
 export const TASKS_SUCCESS = 'TASKS_SUCCESS';
 export const TASKS_REQUEST = 'TASKS_REQUEST';
 export const TASKS_FAILURE = 'TASKS_FAILURE';
```
Comment (MariaAga) [inline webpack/ForemanTasks/Components/TasksTable/TasksTableConstants.js]: These are also not being used, as the api uses TASKS_TABLE_ID for the name and createTableActionTypes for the success,request, failure

File: webpack/ForemanTasks/Components/TasksTable/TasksTableConstants.js [hunk 3]
```diff
@@ -18,4 +18,4 @@ export const CANCEL_SELECTED_CONFIRM_MODAL_ID = 'cancelSelectedConfirmModal';
 export const RESUME_SELECTED_CONFIRM_MODAL_ID = 'resumeSelectedConfirmModal';
 export const UPDATE_CLICKED = 'UPDATE_CLICKED';
 
-export const TASKS_SEARCH_PROPS = getControllerSearchProps('tasks');
+export const TASKS_SEARCH_PROPS = getControllerSearchProps('foreman_tasks/tasks');
```
Comment (MariaAga) [inline webpack/ForemanTasks/Components/TasksTable/TasksTableConstants.js]: This should be:
`export const TASKS_SEARCH_PROPS = {
  ...getControllerSearchProps('tasks'),
  controller: 'foreman_tasks/tasks',
};`
as the controller for the autocomplete is 'tasks' and for the search is 'foreman_tasks/tasks'

**Comment by adamruzicka** [comment]
Thank you @MariaAga for review!

**Pull request merged**